server/src/main/java/org/gluu/oxtrust/action/UpdateOrganizationAction.java

```diff
@@ -174,14 +182,8 @@ private String modifyOrganization()  {
 	}
 
 	private void initOxAuthSetting(){
-		String configurationDn = configurationFactory.getConfigurationDn();
 		try {
-			 ldapOxAuthConfiguration =  organizationService.getOxAuthSetting(configurationDn);
-			 this.webKeysSettings =  ldapOxAuthConfiguration.getOxWebKeysSettings() ;
-			 
-			 if(webKeysSettings == null){
```

Comment:
Probably we need the same in new code

Reply:
I've moved this code to the new WebKeySettingsService